ci/nvrtc_libcudacxx.sh

```diff
@@ -2,18 +2,14 @@
 
 source "$(dirname "$0")/build_common.sh"
 
-CMAKE_OPTIONS="
-    -DCCCL_ENABLE_THRUST=OFF \
-    -DCCCL_ENABLE_LIBCUDACXX=ON \
-    -DCCCL_ENABLE_CUB=OFF \
-    -DCCCL_ENABLE_TESTING=OFF \
-    -DLIBCUDACXX_ENABLE_LIBCUDACXX_TESTS=ON \
-    -DLIBCUDACXX_TEST_WITH_NVRTC=ON \
-"
-configure "$CMAKE_OPTIONS"
+
+PRESET="libcudacxx-nvrtc-cpp${CXX_STANDARD}"
+CMAKE_OPTIONS=""
```

Comment:
We could just add `-DLIBCUDACXX_ENABLE_LIBCUDACXX_TESTS=ON` here instead of the full nvrtc preset